report/src/App.tsx

```diff
@@ -289,15 +317,29 @@ const App = () => {
           <h3 className="font-semibold text-lg mb-1">Notes</h3>
           <ul className="list-disc list-inside">
             <li className="mb-1">
-              All percentages in the table represent API presence whether the shape of the API 
-              matches against the baseline. They are not a calculation of implementation compliance. 
+              All percentages in the table represent whether the
+              shape of the API matches against the baseline. They are not a
+              calculation of implementation compliance.
+            </li>
+            <li className="mb-1">
+              The percentages represent the API surface area that is supported{" "}
+              <span className="font-semibold">or</span> stubbed{" "}
```

Comment:
In the meeting today, we agreed to rename "stub" with "mock" as that's the terminology that unenv uses, so we might as well follow that precedence. Could you please update here and elsewhere in UI and code where we say "stub"? thank you

Reply:
+1 on this